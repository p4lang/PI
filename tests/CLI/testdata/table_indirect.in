act_prof_create_member ecmp_action_profile set_nhop 10.0.0.1 7
table_add ecmp_group 10.0.0.1/16 => 0
act_prof_create_member ecmp_action_profile set_nhop 10.0.0.2 9
table_modify ecmp_group 0 => 1
table_set_default ecmp_group 0
table_dump ecmp_group
table_delete ecmp_group 0
