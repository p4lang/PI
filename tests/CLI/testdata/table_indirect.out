????
Device assigned successfully.
Selecting device.
????
Member was successfully created with handle 0.
????
Entry was successfully added with handle 0.
????
Member was successfully created with handle 1.
????
Entry with handle 0 was successfully modified.
????
Default entry was successfully set.
????
Successfully retrieved 1 entrie(s).
==========
TABLE ENTRIES
**********
Dumping entry 0
Match key:
* ipv4.dstAddr        : LPM       0a000001/16
Indirect handle: 1
==========
Dumping default entry
Indirect handle: 0
==========
????
Entry with handle 0 was successfully removed.
????
